docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
I think this is a windows batch file. A makefile like our other docs would be better imo.